mod-data-import-converter-storage-server/src/test/java/org/folio/rest/impl/snapshot/JobProfileSnapshotTest.java

```diff
@@ -375,25 +372,19 @@ public void shouldReturnSnapshotWrapperForJobProfileWithoutMatchProfileChildWrap
   @Override
   public void clearTables(TestContext context) {
     Async async = context.async();
-    deleteTable(JOB_TO_MATCH_PROFILES)
-      .compose(e -> deleteTable(MATCH_TO_ACTION_PROFILES))
-      .compose(e -> deleteTable(ACTION_TO_MAPPING_PROFILES))
-      .compose(e -> deleteTable(JOB_PROFILES_TABLE))
-      .compose(e -> deleteTable(MATCH_PROFILES_TABLE))
-      .compose(e -> deleteTable(ACTION_PROFILES_TABLE))
-      .compose(e -> deleteTable(MAPPING_PROFILES_TABLE))
-      .onComplete(clearAr -> {
-        if (clearAr.failed()) {
-          context.fail(clearAr.cause());
-        }
-        async.complete();
-      });
-  }
-
-  private Future<RowSet<Row>> deleteTable(String tableName) {
-    Promise<RowSet<Row>> promise = Promise.promise();
     PostgresClient pgClient = PostgresClient.getInstance(vertx, TENANT_ID);
-    pgClient.delete(tableName, new Criterion(), promise);
-    return promise.future();
+    pgClient.delete(JOB_TO_ACTION_PROFILES_TABLE_NAME, new Criterion(), event ->
+      pgClient.delete(JOB_TO_MATCH_PROFILES_TABLE_NAME, new Criterion(), event2 ->
+        pgClient.delete(ACTION_TO_MAPPING_PROFILES_TABLE_NAME, new Criterion(), event3 ->
+          pgClient.delete(MATCH_TO_ACTION_PROFILES_TABLE_NAME, new Criterion(), event4 ->
+            pgClient.delete(JOB_PROFILES_TABLE_NAME, new Criterion(), event5 ->
+              pgClient.delete(MATCH_PROFILES_TABLE_NAME, new Criterion(), event6 ->
+                pgClient.delete(ACTION_PROFILES_TABLE_NAME, new Criterion(), event7 ->
+                  pgClient.delete(MAPPING_PROFILES_TABLE_NAME, new Criterion(), event8 -> {
+                    if (event8.failed()) {
```

Comment:
what the readability advantage of the multilevel nesting depth instead of keeping flat code style?

Reply:
In this case, we`ll have uniformity between all tests.

Reply:
got it, thank you! Just it seems like uniformity can be achieved using flat code style as well